app/assets/stylesheets/views/article.scss

```diff
@@ -35,7 +35,7 @@
         aspect-ratio: auto 650 / 273;
         object-fit: contain;
         border-radius: var(--radius-auto) var(--radius) 0 0;
-        max-height: calc(100vh - var(--header-height) - var(--su-4));
+        max-height: calc(90vh - var(--header-height));
```

Comment:
This is a small eyeball tweak to the max height of image as ongoing QA, related to the original issue.